site/resources.md

```diff
@@ -75,8 +75,8 @@ The end result is a tighter vertical spacing between the opening line of the bul
   _add description_
 
 * Unit-aware operations (Pint)
-
-  _add description_
+* [**Unit-aware operations (Pint)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/units.ipynb)
```

Comment:
Title line needs two spaces at the end (see comment in top of file as well as existing entries for reference).